Workflow/Sources/AnyWorkflow.swift

```diff
@@ -18,17 +18,24 @@
 public struct AnyWorkflow<Rendering, Output> {
     private let storage: AnyStorage
 
+    /// The underlying erased workflow instance
+    public var base: Any { storage.base }
+
     private init(storage: AnyStorage) {
         self.storage = storage
     }
 
     /// Initializes a new type-erased wrapper for the given workflow.
     public init<T: Workflow>(_ workflow: T) where T.Rendering == Rendering, T.Output == Output {
-        self.init(storage: Storage<T>(
-            workflow: workflow,
-            renderingTransform: { $0 },
-            outputTransform: { $0 }
-        ))
+        if let workflow = workflow as? AnyWorkflow<Rendering, Output> {
```

Comment:
is there a way to get the compiler to do this without having to perform the runtime check?